app/pages/index.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const hogan = require('hogan.js');
+
+function renderVerifyIdentityItemPage(vars) {
+    const template = fs.readFileSync(`${__dirname}/templates/verifyIdentityItemTemplate.mustache`, 'utf8');
+    const compiled = hogan.compile(template);
```

Comment:
For performance consideration, 
we can compile the template once, and use it many times?

Reply:
It's a legit concern, however can be thought as improvement.